src/Ledger/GovernanceActions.lagda

```diff
@@ -20,6 +20,7 @@ different purpose.
 
 open import Data.Nat.Properties using (+-0-monoid)
 open import Data.Rational using (ℚ; 0ℚ; 1ℚ)
+open import Data.Product.Base
```

Comment:
Is that import required? I feel like everything in there should already be in scope from the prelude.